subgraphs/uniswap-forks/protocols/camelot-v2/README.md

```diff
@@ -0,0 +1,88 @@
+# Camelot V2 Subgraph
+
+- The code that faciliates the basic operations of this exchange are forked from Uniswap V2. The two main contracts that are forked and used to index data from swaps, deposits, withdraws, and more are the `UniswapV2Factory` and `Pair` contracts.
```

Comment:
Will you include in the README how the camelot subgraph differs from the rest in the Uniswap Forks library?

Reply:
added, thanks 👍